src/OpenQGO.py

```diff
@@ -88,10 +88,16 @@
 from QGONexum import QGOProblem, QGOGraph, QGOOptimizer
 from QGOOut import QGOExporter
 
+from pathlib import Path
+
+DATA_IN_DIR = Path(__file__).resolve().parent.parent / "data_in"
+
 # paths to input data in data_in
-links = ["/home/OpenQGO/data_in/mineral_production_facilities.geojson"]
-links.append("/home/OpenQGO/data_in/mineral_sources.geojson")
-links.append("/home/OpenQGO/data_in/possible_solar.geojson")
+links = [
+    DATA_IN_DIR / "mineral_production_facilities.geojson",
+    DATA_IN_DIR / "mineral_sources.geojson",
+    DATA_IN_DIR / "possible_solar.geojson",
+]

```

Comment:
This makes the file paths relative, so they'll work even outside of the docker container.